system/lib/wasmfs/backends/node_backend.cpp

```diff
@@ -45,7 +45,7 @@ class NodeState {
           result = _wasmfs_node_open(path.c_str(), "r");
           break;
         case O_WRONLY:
-          result = _wasmfs_node_open(path.c_str(), "w");
+          result = _wasmfs_node_open(path.c_str(), "r+");
```

Comment:
```suggestion
          result = _wasmfs_node_open(path.c_str(), "w");
```
or am I missing something?

Reply:
Ah yes, this part at least warrents a comment.

The short answer is that I don't think `O_WRONLY` on its own can be supported (at least not easily).  From the `fopen` man page:

```

       The file descriptor associated with the stream is opened as if by a call to open(2) with the following flags:
              ┌──────────────┬───────────────────────────────┐
              │ fopen() mode │ open() flags                  │
              ├──────────────┼───────────────────────────────┤
              │      r       │ O_RDONLY                      │
              ├──────────────┼───────────────────────────────┤
              │      w       │ O_WRONLY | O_CREAT | O_TRUNC  │
              ├──────────────┼───────────────────────────────┤
              │      a       │ O_WRONLY | O_CREAT | O_APPEND │
              ├──────────────┼───────────────────────────────┤
              │      r+      │ O_RDWR                        │
              ├──────────────┼───────────────────────────────┤
              │      w+      │ O_RDWR | O_CREAT | O_TRUNC    │
              ├──────────────┼───────────────────────────────┤
              │      a+      │ O_RDWR | O_CREAT | O_APPEND   │
              └──────────────┴───────────────────────────────┘
```

Note: there is not way to open write only without either truncation, or appending.    So I chose to just open in read/write mode..

Reply:
Done